aten/src/ATen/cuda/CUDAEvent.h

```diff
@@ -147,8 +147,16 @@ struct TORCH_CUDA_CPP_API CUDAEvent {
 
   // Note: cudaEventElapsedTime can be safely called from any device
   float elapsed_time(const CUDAEvent& other) const {
-    TORCH_CHECK(is_created_ && other.isCreated(),
-      "Both events must be recorded before calculating elapsed time.");
+    TORCH_CHECK(
```

Comment:
Shouldn't these mostly be ValueError? Use TORCH_CHECK_VALUE

Reply:
Thank you ,have update it.